fusesoc/vlnv.py

```diff
@@ -92,17 +92,12 @@ def _is_version(s):
         self.sanitized_name = str(self).lstrip(":").replace(":", "_")
 
     def __str__(self):
-        if self.relation == "=":
-            relation = ""
-        else:
-            relation = self.relation
```

Comment:
It looks to me like this would no longer print e.g. `>=vendor:library:name:version` but maybe this isn't used anywhere?

Reply:
Ok, looked at the code now. Yeah, it's dead.